content/intro-to-storybook/react/en/using-addons.md

```diff
@@ -73,6 +73,16 @@ import { action } from '@storybook/addon-actions';
 import { withKnobs, object } from '@storybook/addon-knobs/react';
 ```
 
+If using typescript
+
+```typescript
+// src/components/Task.stories.tsx
+
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
```

Comment:
@ron4stoppable instead of adding the snippet we probably could do better and go with the following:

``` html
<div class="aside">If you're using TypeScript, you'll need to make a small adjustment to the imports. You'll need to use <code>import { withKnobs, object } from '@storybook/addon-knobs'</code> instead.</div>
```

Reasoning behind this is to avoid introducing typescript in the middle of the chapter. As the flow and code usage in this version of the tutorial leans towards standard javascript.

Reply:
This is a good compromise.

Reply:
sure thing @jonniebigodes, I was feeling doubtful adding the sample redundancy.